data/editor.config

```diff
@@ -555,7 +561,7 @@
       <type id="natural-hot_spring">
         <include field="name" />
       </type>
-      <type id="natural-peak" can_add="no">
+      <type id="natural-peak">
```

Comment:
Are there many not-mapped peaks?

Reply:
![image](https://github.com/organicmaps/organicmaps/assets/18434508/a6272751-f5fc-4ce2-b10d-e09499ba03d9)